controllers/controller_common.go

```diff
@@ -118,3 +122,23 @@ func applyClusterRefLabel(cluster *v1beta1.KafkaCluster, labels map[string]strin
 func SetNewKafkaFromCluster(f func(k8sclient client.Client, cluster *v1beta1.KafkaCluster) (kafkaclient.KafkaClient, func(), error)) {
 	newKafkaFromCluster = f
 }
+
+// SkipClusterRegistryOwnedResourcePredicate returns controller event filter that filters
+// out events triggered by Cluster Registry owned resources
+type SkipClusterRegistryOwnedResourcePredicate struct{}
+
+func (SkipClusterRegistryOwnedResourcePredicate) Create(e event.CreateEvent) bool {
+	return !util.ObjectManagedByClusterRegistry(e.Object)
```

Comment:
Do we still need to do `object, err := meta.Accessor(e.Object)` before checking if the object is managed by cluster registry?